downloads/index.md

```diff
@@ -179,7 +179,7 @@ Checksums for this release are available in both, [MD5](https://julialang-s3.jul
       <td colspan="3"> </td>
     </tr>
     <tr>
-      <th> macOS ARM <a href="/downloads/platform/#macos">[help]</a></th>
+      <th> macOS ARM (M1 Mac)<a href="/downloads/platform/#macos">[help]</a></th>
```

Comment:
Maybe
```suggestion
      <th> macOS ARM (M1 processor)<a href="/downloads/platform/#macos">[help]</a></th>
```
?

Reply:
I think "M-series processor" might be good as well since one can imagine an M2 someday soon.

Reply:
You could also consider "Apple Silicon".